src/main.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
-import App from 'src/App.jsx';
+import App from './App.jsx';
```

Comment:
I dont know why the absolute path does not work on your side, but on my side it works without problem. I think its better to change it back to the original absolute path.